.docker/entrypoint/Gemfile.lock

```diff
@@ -112,13 +114,14 @@ GEM
     octokit (4.18.0)
       faraday (>= 0.9)
       sawyer (~> 0.8.0, >= 0.5.3)
+    open3 (0.1.0)
     parallel (1.19.2)
     parser (2.7.2.0)
       ast (~> 2.4.1)
     pathutil (0.16.2)
       forwardable-extended (~> 2.6)
     public_suffix (4.0.6)
-    rack (2.1.4)
+    rack (1.6.13)
```

Comment:
Codacy found an issue: [Directory traversal in Rack::Directory app bundled with Rack (CVE-2020-8161)](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6320163)